src/main/java/seedu/address/logic/Messages.java

```diff
@@ -26,9 +26,9 @@ public class Messages {
                 "Oops! Multiple values are specified for the following single-valued parameter(s): ";
 
     public static final String MESSAGE_NON_INTEGER_INDEX = "Oops! The index you provided is not valid.\n"
-            + "It appears to be either not a valid integer or it exceeds the maximum allowed value "
-            + "(2147483647).\n"
-            + "Please try again with a valid positive integer!";
+            + "It should be a positive integer and within the range of 1 to "
```

Comment:
The case of not an integer and the case for not in range from 1 to Integer.MAX_VALUE should be split into two seperate cases.

Reply:
Not an integer check should take precedence over not in range check

Reply:
Also what are we doing for these cases:
https://byjus.com/us/math/the-different-forms-of-representing-numbers/#:~:text=The%20three%20forms%20to%20represent,%2C%20500%20%2B%2020%20%2B%203.

1. `100          000` [Spaces format]
2. `100,000` [Standard Form] 
3.  `three hundred ninety-nine` [Word Form]

Reply:
> The case of not an integer and the case for not in range from 1 to Integer.MAX_VALUE should be split into two seperate cases.
> 
> Possible to get flagged for potential feature flaw for non-specific error message.

When it comes to telling the user what went wrong with their input index, distinguishing between negative and nonstring indexes provides no additional value.

Reply:
Alright if you say so, will approve for now ...